src/Exception/UnexpectedValueException.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace Http\Client\Exception;
-
-use Http\Client\Exception;
-
-/**
- * Custom Unexpected Value exception
- *
- * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
- */
-final class UnexpectedValueException extends \UnexpectedValueException implements Exception
-{
```

Comment:
removing this means a user can not catch this specific exception anymore. are there valid use cases to want catch this one?